data/deprecated outfits.txt

```diff
@@ -130,15 +130,6 @@ outfit "Reverse Thruster"
 	"reverse thrusting energy" 1.8
 	"reverse thrusting heat" 3.5
 
-# Deprecated because it unbalances the capture dynamics, among other things.
-outfit "NDR-114 Android"
```

Comment:
I'd prefer that we give the new android outfit a different name as opposed to reusing the old one, even if it's just as simple as naming the new one "NDR-115 Android." If some mad lad out there has actually been holding onto NDR-114s for all these years, I say let them keep them untouched.

Reply:
I don't recall when it was - probably a long time - I seem to recall someone saying that this deprecated category was only intended as a safety mechanism while the game was in beta and thus changing a lot. A way to make things friendlier when stuff was changing every few months. As such, it was going to be purged when the game reached 1.0.0. After all, the entire deprecated list is basically the recycling bin for the project: Stuff we want to delete/remove but can't currently because it might break someone's save.

I didn't choose to re-use the android outfit because I liked the outfit image.  Nor did I do it because I particularly liked the name. I chose it because:
a) It's a historical artifact in game that echoes the fact it is a  historical reference to our game development.
b) It makes it a legitimate and balanced part of the game instead of a supposedly OP thing that people have to roll-back their game version to get and/or save-edit for.
c) It means that - unlike other depreciated stuff - this will keep on existing, albeit in a less powerful way, indefinitely into the future.

A is the big one, because of the thematics of the mission. The Remnant are literally pulling out historical artifacts; and for players that have been around since those early days of ES' development, I hope to generate some of the same feeling in them. This is best achieved by having it be specifically as close to the exact same thing as possible. Some changes are required due to balance, of course; but I don't see any need to let people keep supposedly broken and OP outfits just because.

Reply:
> As such, it was going to be purged when the game reached 1.0.0.
> It means that - unlike other depreciated stuff - this will keep on existing...

I think we can cross that bridge when we get there.

> It's a historical artifact in game that echoes the fact it is a historical reference to our game development.

One can echo a historical reference to the game's development without using the exact same name.

> It makes it a legitimate and balanced part of the game instead of a supposedly OP thing that people have to roll-back their game version to get and/or save-edit for.

Someone could also just create a plugin that adds un-nerfed androids, so I don't think that people save editing or rolling back their game version to acquire the old androids should be of concern.

Reply:
> I think we can cross that bridge when we get there.

Or we can just solve it here and now, never to be a concern again.

> One can echo a historical reference to the game's development without using the exact same name.

That is true, however it is more impactful to have the old thing brought back into canonicity than to have a new thing that just happens to have a similar name and re-uses the image.

> Someone could also just create a plugin that adds un-nerfed androids, so I don't think that people save editing or rolling back their game version to acquire the old androids should be of concern.

Agreed. So let's just change these ones, and anyone who really wants the un-nerfed original androids can add them back in or un-nerf then to their hearts content.